module-api/src/main/java/com/yapp/sharefood/user/controller/UserController.java

```diff
@@ -25,7 +29,44 @@
     public ResponseEntity<UserNicknameResponseDto> findNotExistNickName() {
         String newNickname = userService.createUniqueNickname();
         UserNicknameResponseDto userNicknameResponseDto = new UserNicknameResponseDto(newNickname);
-
         return ResponseEntity.ok(userNicknameResponseDto);
     }
+
+    @GetMapping("/api/v1/users/nickname/check")
```

Comment:
check는 동사기 때문에 rest api 규약에 맞지 않습니다.
validation이 맞을거 같습니다
userId도 빠진거 같습니다.
```suggestion
    @GetMapping("/api/v1/users/{userId}/nickname/validation")
```

Reply:
@kihwankim
기환님 저 닉네임 validation하는건 꼭 특정 유저여야할까요? 회원가입의 경우에도 쓸 수 있을거같은데 userId를 넣어야하는 이유가 있을까요?

Reply:
이건 회원가입 용도가 아닌 회원 정보 변경 이라서 저장된 특정 user 기 때문에 userid 가 있엉ㅑ합니다